src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/SendSlackNotification.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.Types.PaymentProcess;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess
+{
+    public class SendSlackNotification
+    {
+        private readonly ICommandDispatcher _commandDispatcher;
+
+        public SendSlackNotification(ICommandDispatcher commandDispatcher)
+        {
+            _commandDispatcher = commandDispatcher;
+        }
+
+        [FunctionName(nameof(SendSlackNotification))]
+        public async Task Complete([ActivityTrigger] SendSlackNotificationInput input)
+        {
+            try
+            {
+                var command = new SlackNotificationCommand(input.Message);
+                await _commandDispatcher.Send(command);
+            }
+            catch { }
```

Comment:
If we don't want to stop processing on failure, should we log that an error has occurred sending to Slack?